app/records/template_tags/records_tags.py

```diff
@@ -0,0 +1,6 @@
+from app.records.field_labels import FIELD_LABELS
+
+
+def as_label(record_field_name: str) -> str:
```

Comment:
Instead of `as_label`, could we call this something more specific like `record_field_label`?